package.json

```diff
@@ -48,6 +48,7 @@
     "kss": "^3.0.1",
     "npm": "^7.16.0",
     "path": "^0.12.7",
+    "sass": "^1.52.1",
```

Comment:
Could we maybe then use directly : `const sass = require('sass');` as in their docs :thinking: but looks good to me anyway like this